apps/keyboard/js/layouts/de.js

```diff
@@ -25,7 +25,7 @@ Keyboards.de = {
     ], [
       { value: 'a' }, { value: 's' }, { value: 'd' }, { value: 'f' },
       { value: 'g' } , { value: 'h' }, { value: 'j' }, { value: 'k' },
```

Comment:
extra space before `,`, this is not related to your changes, but you could help fix them if you want to.

Thanks.